webapp/views.py

```diff
@@ -937,7 +937,11 @@ def marketo_submit():
     visitor_data = {
         "userAgentString": flask.request.headers.get("User-Agent"),
     }
-    referrer = flask.request.referrer
+    referrer = (
+        flask.request.referrer
+        if flask.request.referrer
+        else "https://ubuntu.com"
```

Comment:
How did you came up with u.com for this?
Also if you would like you could also do same in one liner like `referrer = flask.request.referrer or "https://ubuntu.com"` (optional)

Reply:
This comes from Lazlo in GE https://chat.canonical.com/canonical/pl/z5b1b9fh6inijmpjqraaxnz51a